packages/frontend-2/components/dashboard/Sidebar.vue

```diff
@@ -183,12 +216,6 @@ const projectsLink = computed(() => {
     : projectsRoute
 })
 
-const showSidebar = computed(() => {
```

Comment:
Everybody can now see the sidebar, as long as you are logged in, just hiding the "projects" link if you are not part of the active workspace